TARGETS

```diff
@@ -1148,9 +1148,9 @@ ROCKS_TESTS = [
         parallelism = parallelism,
         rocksdb_arch_preprocessor_flags = ROCKSDB_ARCH_PREPROCESSOR_FLAGS,
         rocksdb_compiler_flags = ROCKSDB_COMPILER_FLAGS,
-        rocksdb_external_deps = ROCKSDB_EXTERNAL_DEPS,
         rocksdb_os_deps = ROCKSDB_OS_DEPS,
         rocksdb_os_preprocessor_flags = ROCKSDB_OS_PREPROCESSOR_FLAGS,
+        rocksdb_external_deps = ROCKSDB_EXTERNAL_DEPS,
```

Comment:
Not big deal but before the change, it seems to follow alphabet order. Right now what order is it?

Reply:
Good point. Fixed.